pkg/cloudprovider/provider/vultr/provider.go

```diff
@@ -50,11 +65,17 @@ func New(configVarResolver *providerconfig.ConfigVarResolver) cloudprovidertypes
 }
 
 type Config struct {
-	APIKey string
-	Region string
-	Plan   string
-	OsID   string
-	Tags   []string
+	PhysicalMachine bool
+	APIKey          string
+	Region          string
+	Plan            string
+	OsID            string
+	Tags            []string
+	VpcID           []string
+	EnableVPC       bool
+	EnableIPv6      bool
+	EnableVPC2      bool
+	Vpc2Id          []string
```

Comment:
```suggestion
	Vpc2ID          []string
```

To make sure it was an intentional difference with other cases where `ID` used uppercase. Would it make sense to follow here the same style?